### DISCUSSION TITLE
Bot agents does not work

### DISCUSSION THREAD

**Discussion by igmer** [chatwoot/chatwoot#6334] [Chatwoot APIs] [answered]
I have added an Agent bot but it does not respond, the request is made correctly to the webhook but the response is not processed by chatwoot, do I need to do an extra step?

**Comment by lucianodltec** [comment]
Hi @igmer , did you get any solution for this? I'm facing the same issue.

**Comment by sojan-official** [comment, marked as answer]
@igmer the idea is that the bot should send a response back to chatwoot via the message APIs.  We don't process the response from the webhooks.
